src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,13 @@ public CommandResult execute() {
         }
     }
 
+    /**
+     * Determines if execution of commands mutates the address book
+     *
+     * @return true, add command mutates data
```

Comment:
Good job on remembering to add the heading comments for `isMutating()` methods and also specifying the return type!